pom.xml

```diff
@@ -15,7 +15,7 @@
   <parent>
     <groupId>org.terasoluna.gfw</groupId>
     <artifactId>terasoluna-gfw-parent</artifactId>
-    <version>5.5.0-SNAPSHOT</version>
+    <version>5.5.0.RC2-SNAPSHOT</version>
```

Comment:
- [ ] Please also change branch name at [travis.yml](https://github.com/terasolunaorg/terasoluna-gfw-functionaltest/blob/c5b371ecf0d0faad6d61544213b1c7a004bf8b54/.travis.yml#L15).

Reply:
I checked to fix at 9f85b26 , thanks @btkobayashirun .